03_letters/anishaNair/main_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := `a : 2
+b : 1
+`
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+func TestCountLettersOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// Then
+	expected := map[rune]int{104: 1, 97: 2, 110: 1, 105: 1, 115: 1}
+	actual := countLetters("anisha")
+	r.Equalf(expected, actual, "Unexpected output in main()")
```

Comment:
Bad failure message.
Write a few more test cases including empty strings and unicodes with [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)